backend/dataall/modules/omics/services/omics_permissions.py

```diff
@@ -0,0 +1,43 @@
+"""
+Add module's permissions to the global permissions.
+Contains permissions for Omics RUNs
+"""
+
+from dataall.core.permissions.services.environment_permissions import (
+    ENVIRONMENT_INVITED,
+    ENVIRONMENT_INVITATION_REQUEST,
+    ENVIRONMENT_ALL,
+)
+from dataall.core.permissions.services.resources_permissions import (
+    RESOURCES_ALL_WITH_DESC,
+    RESOURCES_ALL,
+)
+from dataall.core.permissions.services.tenant_permissions import TENANT_ALL, TENANT_ALL_WITH_DESC
+
```

Comment:
UPDATE_OMICS_RUN is not used, so we should remove it

Reply:
Fixed! https://github.com/data-dot-all/dataall/pull/954/commits/82e997c20af69ad9fbae37ca312644dc07f68929